build.sbt

```diff
@@ -18,9 +18,9 @@ lazy val V = new {
   val specs2: String           = "4.5.1"
   val enumeratum: String       = "1.5.13"
   val enumeratumCirce: String  = "1.5.21"
```

Comment:
Would you need to bump up enumeratumCirce?

Reply:
`enumeratum-circe` is already on the latest version